rmgpy/tools/canteramodel.py

```diff
@@ -63,18 +64,45 @@ class CanteraCondition(object):
 
     """
 
-    def __init__(self, reactor_type, reaction_time, mol_frac, T0=None, P0=None, V0=None):
+    def __init__(self, reactor_type, reaction_time, mol_frac, surface_mol_frac=None, T0=None, P0=None, V0=None):
         self.reactor_type = reactor_type
         self.reaction_time = Quantity(reaction_time)
 
         # Normalize initialMolFrac if not already done:
         if sum(mol_frac.values()) != 1.00:
+            logging.warning('Initial mole fractions do not sum to one; normalizing.')
+            logging.info('')
```

Comment:
This is a "super-nitpick", but could we change the next line to be '\nOriginal Composition' instead of logging an empty line to get the newline? If this is done like this elsewhere please ignore this comment.

Reply:
Yeah, I copied this from line 357 (and 478) of rmgpy/rmg/input.py, so I'll leave it for now.